apps/api/src/migration/1662066445229-UpdateAtsApplicantIdToUUID.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner, TableColumn } from "typeorm"
+
+export class UpdateAtsApplicantIdToUUID1662066445229 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.changeColumn('ien_applicants','applicant_id',new TableColumn({
+            type:'uuid',
+            name:'applicant_id',
+            isNullable:true
+        }) ) 
+
+        await queryRunner.changeColumn('ien_applicants','status_id',new TableColumn({
+            type:'uuid',
```

Comment:
Does ATS change its status id to `uuid`? I don't think we need `uuid` for small, definite number of items.